doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -477,6 +477,7 @@ system.
 | ------------------- | ---------------- | --------                                                                                                       |
 | `id`                | [`text`]         | The ID of the source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).                   |
 | `replication_slot`  | [`text`]         | The name of the replication slot in the PostgreSQL database that Materialize will create and stream data from. |
+| `timeline_id`       | [`uint8`]        | The PostgreSQL timeline id determined when creating the source. Used to detect backup restores.                |
```

Comment:
I would shy away from explaining what we use the timeline ID for internally. These are reference docs; we only need to explain what these things _are_, not how to use them. If anything I'd link to whatever PostgreSQL docs exist on what a timeline ID is.

```suggestion
| `timeline_id`       | [`uint8`]        | The PostgreSQL timeline ID determined when creating the source.                                                |
```

Reply:
sounds good!